src/services/chris_integration.tsx

```diff
@@ -238,119 +238,164 @@ class ChrisIntegration {
+          }
+  
+          for (let fileObj of pluginInstanceFiles.getItems()) {
+            if (fileObj.data.fname.includes('prediction') && fileObj.data.fname.includes('json')) {
+              let content = await this.fetchJsonFiles(fileObj.data.id);
+              const formatNumber = (num: any) => (Math.round(Number(num) * 10000) / 100) // to round to 2 decimal place percentage
+              Object.keys(content).map(function(key: string) { // Reading in the classifcation titles and values
+                if ((key !== 'prediction') && (key !== 'Prediction')) {
+                  if ((key !== '**DISCLAIMER**') && (!isNaN(content[key]))) {
+                    newSeries.classifications.set(key, formatNumber(content[key]));
+                  }
                 }
+              });
+  
+            } else if (fileObj.data.fname.includes('severity.json')) {
+              let content = await this.fetchJsonFiles(fileObj.data.id)
+              newSeries.geographic = {
+                severity: content['Geographic severity'],
+                extentScore: content['Geographic extent score']
               }
-            });
-
-          } else if (fileObj.data.fname.includes('severity.json')) {
-            let content = await this.fetchJsonFiles(fileObj.data.id)
-            newSeries.geographic = {
-              severity: content['Geographic severity'],
-              extentScore: content['Geographic extent score']
-            }
-            newSeries.opacity = {
-              severity: content['Opacity severity'],
-              extentScore: content['Opacity extent score']
-            }
-          } else if (!fileObj.data.fname.includes('json')) {
-            newSeries.imageId = fileObj.data.id;
-            
-            // Fetch image URL
-            if (newSeries.imageId) {
-              const imgBlob = await DicomViewerService.fetchImageFile(newSeries.imageId);
-              const urlCreator = window.URL || window.webkitURL;
-              newSeries.imageUrl = urlCreator.createObjectURL(imgBlob);
+              newSeries.opacity = {
+                severity: content['Opacity severity'],
+                extentScore: content['Opacity extent score']
+              }
+            } else if (!fileObj.data.fname.includes('json')) {
+              newSeries.imageId = fileObj.data.id;
+              
+              // Fetch image URL
+              if (newSeries.imageId) {
+                const imgBlob = await DicomViewerService.fetchImageFile(newSeries.imageId);
+                const urlCreator = window.URL || window.webkitURL;
+                newSeries.imageUrl = urlCreator.createObjectURL(imgBlob);
+              }
+  
+              // get dcmImageId from dircopy
+              const dircopyPlugin = pluginlists[pluginlists.findIndex((plugin: any) => plugin.data.plugin_name === PluginModels.Plugins.FS_PLUGIN)]
+              const dircopyFiles = (await dircopyPlugin.getFiles({
+                limit: 100,
```

Comment:
Would these limits and offsets be related to the story at hand and this task? WOuld they have to be updated or changed, due to your changes?